src/new.rs

```diff
@@ -19,11 +19,22 @@ pub struct New {
     /// How many accounts to cache by default (Default 10)
     #[clap(short, long)]
     pub cache_accounts: Option<usize>,
+
+    /// Directly provide the wallet password when create new wallet.
+    ///
+    /// WARNING: This is primarily provided for non-interactive testing. Using this flag is
+    /// prone to leaving your password exposed in your shell command history!
+    #[clap(short, long)]
+    pub unsafe_password: Option<String>,
```

Comment:
I don't agree that this should be a command line argument. I think we should support an environment variable, but a CLI argument is too easily leaked.